paws_collector.js

```diff
@@ -37,11 +37,16 @@ const ERROR_CODE_DUPLICATE_STATE = 'DUPLICATE_STATE';
 const ERROR_CODE_COMPLETED_STATE = 'COMPLETED_STATE';
 const SQS_VISIBILITY_TIMEOUT = 900;
 const DDB_TTL_DAYS = 14;
-const DEDUP_LOG_TTL_HOURS = 24;
+const DEDUP_LOG_TTL_SECONDS = 86400;
 const DDB_OPTIONS = {
     maxRetries: 10,
     ConsistentRead: true
 };
+
+const DDB_DELETE_BATCH_OPTIONS = {
+    maxBatchSize: 25,
+    maxBatchSizeBytes: 16 * 1024 * 1024
+}
```

Comment:
Are we missing a semi-colon here? Just for the sake of consistency and placating the linter